setup.cfg

```diff
@@ -20,6 +20,7 @@ install_requires =
     more-itertools>=3.1.0
     psutil>=5.4.0
     requests>=2.19.1
+    SQLAlchemy>=1.4
```

Comment:
Just to make sure: 1.4 works with this, given the title of the pr?

Reply:
It was implemented as a breaking change in 2.0, but has a backport since 1.4 to support a smooth transition. You will need at least 1.4 to use it, but in 2.0 it will break...

Reply:
/Actually, it does not break somehow but I want to get rid of this warning and avoid problems with further 2.x versions/